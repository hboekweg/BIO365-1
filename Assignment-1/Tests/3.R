source("3.R")
