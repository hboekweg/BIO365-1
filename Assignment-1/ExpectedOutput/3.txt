[1] "8460.2"
[1] 8460.2
[1] 8460
[1] TRUE
